src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,19 @@
 package core.basesyntax;
 
 public class Application {
+    private static final int NUMBER_OF_BALLS = 3;
+    private static final Lottery lottery1 = new Lottery();
+
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery[] lottery = new Lottery[NUMBER_OF_BALLS];
+        for (int i = 0; i < lottery.length; i++) {
+            lottery[i] = lottery1.getRandomBall();
+            printResult(lottery[i]);
```

Comment:
The same way here. Call toString() on lottery[i]

Reply:
fixed